InnerEye/ML/utils/run_recovery.py

```diff
@@ -63,25 +63,23 @@ def download_best_checkpoints_from_child_runs(config: OutputParams, run: Run) ->
     @staticmethod
     def download_all_checkpoints_from_run(config: OutputParams, run: Run) -> RunRecovery:
         """
-        Downloads all checkpoints of the provided run: The best checkpoint and the recovery checkpoint.
-        A single folder inside the checkpoints folder will be created that contains the downloaded checkpoints.
+        Downloads all checkpoints of the provided run inside the checkpoints folder.
```

Comment:
This is now potentially a rather expensive operations: We are downloading top_k checkpoints + best checkpoints, rather than 2. You could probably reduce that to only getting the single "latest recovery" checkpoint - you have the logic already to get the "latest" checkpoint

Reply:
I would rather not limit the download here, because for example for SSL logic i need to be able to tell in a custom Container argument which checkpoint to use. If we don't have to possibly to use other checkpoints it would defeat the purpose of allowing multiple checkpoints to be saved in my opinion